zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -480,7 +496,18 @@ impl<F: Field> SignVerifyChip<F> {
         copy(ctx, "is_address_zero", a, is_address_zero)?;
         copy(ctx, "pk_rlc", config.rlc, pk_rlc)?;
         ctx.next();
-        copy(ctx, "pk_hash_rlc", config.rlc, pk_hash_rlc)?;
```

Comment:
line !498 ctx.next() should be remove since word_lo, word_hi are in the same row

Reply:
fixed in [9cc1377](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1418/commits/9cc137741df07065f05232a6f9d6534290832af0)